doozer/doozerlib/backend/konflux_image_builder.py

```diff
@@ -273,6 +274,8 @@ def _new_pipelinerun(generate_name: str, application_name: str, component_name:
         for param in obj["spec"]["params"]:
             if param["name"] == "output-image":
                 param["value"] = output_image
+            if param["name"] == "skip-checks":
+                param["value"] = "true"
```

Comment:
@vfreex this doesn't seem to be working, not sure if you have any idea?

Reply:
ship-checks parameter is not defined under `spec.params` (see line 19 of the yaml template). We can add it as in https://github.com/openshift-eng/art-tools/pull/955/files#diff-e45d70130344e3d45bd59cb091bd0a794121c675ce90f17c5b4e4702ded620ffR281

Reply:
Thanks!